bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller.go

```diff
@@ -1041,11 +1041,14 @@ func (r *KubeadmConfigReconciler) ensureBootstrapSecretOwnersRef(ctx context.Con
 	if err != nil {
 		return errors.Wrapf(err, "failed to add KubeadmConfig %s as ownerReference to bootstrap Secret %s", scope.ConfigOwner.GetName(), secret.GetName())
```

Comment:
Let's please update the func godoc as well ("has the configOwner as a controller OwnerReference")